src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -43,6 +43,7 @@
         <service id="form.type_guesser.validator" class="Symfony\Component\Form\Extension\Validator\ValidatorTypeGuesser">
             <tag name="form.type_guesser" />
             <argument type="service" id="validator.mapping.class_metadata_factory" />
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
@sstok @Taluu Thanks for the review :)

Are you sure, I actually did not copy it from yams :) I did it because tests were failing because of that :/